### PR TITLE
ffmpeg: update page

### PR DESCRIPTION
**Pull request by o0nd7ots**
I think that we have enough video guidance and converting audio is also needed.

I have added conversion to Red Book CD format.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/common/ffmpeg.md [hunk 1]
```diff
@@ -7,6 +7,10 @@
 
 `ffmpeg -i {{path/to/video.mp4}} -vn {{path/to/sound.mp3}}`
 
+- Transcode any FLAC file to Red Book CD format (44100kHz, 16bit)
```
Comment (fazlearefin) [inline pages/common/ffmpeg.md:10]:
```suggestion
- Transcode any FLAC file to Red Book CD format (44100kHz, 16bit):
```

File: pages/common/ffmpeg.md [hunk 2]
```diff
@@ -7,6 +7,10 @@
 
 `ffmpeg -i {{path/to/video.mp4}} -vn {{path/to/sound.mp3}}`
 
+- Transcode any FLAC file to Red Book CD format (44100kHz, 16bit):
+
+`ffmpeg -i audio_file.flac -ar 44100 -sample_fmt s16 audio_file.wav`
```
Comment (kbdharun) [inline pages/common/ffmpeg.md:12]:
```suggestion
`ffmpeg -i {{path/to/input_audio.flac}} -ar 44100 -sample_fmt s16 {{path/to/output_audio.wav}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have a minor suggestion for this page.

We could convert the placeholders to use proper path placeholders to stay consistent with the other examples.

**Review by Magrid0 — APPROVED** [review]
Looks good to me with the small correction kbdharun made

File: pages/common/ffmpeg.md [hunk 3]
```diff
@@ -7,6 +7,10 @@
 
 `ffmpeg -i {{path/to/video.mp4}} -vn {{path/to/sound.mp3}}`
 
+- Transcode any FLAC file to Red Book CD format (44100kHz, 16bit):
```
Comment (acuteenvy) [inline pages/common/ffmpeg.md:10]:
```suggestion
- Transcode a FLAC file to Red Book CD format (44100kHz, 16bit):
```

**Review by kbdharun — APPROVED** [review]

**Pull request merged**